mm2src/coins/eth.rs

```diff
@@ -6576,15 +6576,15 @@ pub fn checksum_address(addr: &str) -> String {
 
 /// `eth_addr_to_hex` converts Address to hex format.
 /// Note: the result will be in lowercase.
-pub fn eth_addr_to_hex(address: &Address) -> String { format!("{:#02x}", address) }
+fn eth_addr_to_hex(address: &Address) -> String { format!("{:#x}", address) }
 
 /// Checks that input is valid mixed-case checksum form address
 /// The input must be 0x prefixed hex string
 fn is_valid_checksum_addr(addr: &str) -> bool { addr == checksum_address(addr) }
 
 /// `display_eth_address` converts Address to mixed-case checksum form.
 #[inline]
-pub fn display_eth_address(addr: &Address) -> String { checksum_address(&eth_addr_to_hex(addr)) }
+pub fn display_eth_address(addr: &Address) -> String { checksum_address(&addr.addr_to_string()) }
```

Comment:
Do you not like to make this also a trait?
(I think we generally should try to reduce quantity of functions in the code)

Reply:
Found out we already have DisplayAddress trait, so reutilized it https://github.com/KomodoPlatform/komodo-defi-framework/pull/2348/commits/3556fd6d4985c2792b8e4c14e027909a9b7eaf3f